README.md

```diff
@@ -100,17 +100,11 @@ npm run test-live
 ## Run end-to-end tests
 In order to run e2e tests you need to install [lisk-core](https://github.com/LiskHQ/lisk)
 
-#### Setup
+#### Setup core
 
-Setup protractor
-
-```
-./node_modules/protractor/bin/webdriver-manager update
-```
-
-Setup a lisk test node to run on localhost:4000 as described in https://github.com/LiskHQ/lisk#tests
-
-And run it with [pm2](http://pm2.keymetrics.io/).
+1) Setup a lisk test node as described in [https://github.com/LiskHQ/lisk#tests](https://github.com/LiskHQ/lisk#tests)
+2) Apply blockchain snapshot `./e2e-test-setup.sh ~/git/lisk/` (replace `~/git/lisk/` with your path to lisk core)
```

Comment:
I think `./e2e-test-setup.sh ~/git/lisk/`  should be in the "Run" section, because it's needed every time before running the e2e suite of tests